crds_data.ipynb

```diff
@@ -533,6 +533,95 @@
     "df.head()\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Apply logbook flags"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def read_logbook(path):\n",
+    "    \"\"\"\n",
+    "    Read events in the logbook file (.csv) \n",
+    "    Drop empty lines.\n",
+    "    \n",
+    "    Dataframe columns name: Event | Start_Event | End_Event | Flags\n",
+    "    \n",
+    "    Parameters: path of .csv logbook\n",
+    "    Returns: dataframe with the logbook data\n",
+    "    \"\"\"\n",
+    "    lines = open(path_logbook).readlines()\n",
+    "    lines_skip = lines.index('Event,\"Start Event\\n')\n",
+    "    df = pd.read_csv(path, sep=',',  skiprows=lines_skip)\n",
+    "    df = df.rename(columns={\"Start Event\\ndd/mm/aaaa hh:mm\": \"Start_Event\",\n",
+    "                           \"End Event\\ndd/mm/aaaa hh:mm\": \"End_Event\"})\n",
```

Comment:
We can drop this lines using the new .csv file

Reply:
Done!